File: sandbox/kortix-master/tests/fixtures/test-scripts/test-env.py
#!/usr/bin/env python3
import os
import sys
import json

def main():
    if len(sys.argv) < 2:
        print(json.dumps({"error": "Key argument required"}))
        sys.exit(1)
    
    key = sys.argv[1]
    value = os.environ.get(key)
    found = key in os.environ
    
    result = {
        "language": "python",
        "key": key,
        "value": value,
        "found": found
    }
    
    print(json.dumps(result))

if __name__ == "__main__":
    main()

File: sandbox/kortix-master/tests/fixtures/test-scripts/test-env.go
package main

import (
	"encoding/json"
	"os"
)

type Result struct {
	Language string `json:"language"`
	Key      string `json:"key"`
	Value    string `json:"value"`
	Found    bool   `json:"found"`
}

func main() {
	if len(os.Args) < 2 {
		result := map[string]string{"error": "Key argument required"}
		json.NewEncoder(os.Stdout).Encode(result)
		os.Exit(1)
	}

	key := os.Args[1]
	value, found := os.LookupEnv(key)

	result := Result{
		Language: "go",
		Key:      key,
		Value:    value,
		Found:    found,
	}

	json.NewEncoder(os.Stdout).Encode(result)
}
